test/test.js

```diff
@@ -495,6 +510,44 @@ tape('Non-JSON logs may carry timestamp.', function (t) {
   logger[lvl]('test');
 });
 
+tape('Non-JSON logs may carry Hostname.', function (t) {
+  t.plan(1);
+  t.timeoutAfter(2000);
+
+  mockTest(function (data) {
+    t.true(pattern.test(data), 'matched');
+
+  });
+  const os = require('os');
+  const lvl = defaults.levels[3];
+  const tkn = token;
+  const pattern = new RegExp('^' + token +' ' + os.hostname() + ' \\w+ test\\n$'
+  );
+
+  const logger = new Logger({ token: tkn, withHostname: true, region: 'eu' });
+
+  logger[lvl]('test');
+});
+
+
+tape('JSON logs may carry Hostname.', function (t) {
+  t.plan(1);
+  t.timeoutAfter(2000);
+
+  mockTest(function (data) {
+    const log = JSON.parse(data.substr(37));
+    t.true(log.host, 'has property');
```

Comment:
can we assert the hostname here?

```suggestion
    t.equals(os.hostname(), log.host);
```